### DISCUSSION TITLE
Difficulty Beginning Copilot Subscription After Post-Beta Trial Expires

### DISCUSSION THREAD

**Discussion by githubpsyche** [community/community#43567] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Hi,

I had Github Copilot during the free beta and afterwarrds was given a 60 day trial. This trial has expired. I'd like to pay to continue using github copilot. 

There are two places I've visited in an attempt to do this: the [Copilot settings page](https://github.com/settings/copilot) of my personal account and the [Billing settings](https://github.com/settings/billing) page of my personal account. Each of these is mentioned in the page "[Managing your GitHub Copilot for Individuals subscription](https://docs.github.com/en/billing/managing-billing-for-github-copilot/managing-your-github-copilot-for-individuals-subscription)" of the Github Copilot section of Github Docs. 

But at both locations, the option to update my Github Copilot subscription is missing. There is no dropdown box for changing to yearly billing, for example:

![image](https://user-images.githubusercontent.com/33912594/211154553-2e5150bd-548a-4ef4-bba6-709763d19196.png)

I also can't find a way to disable or remove Github Copilot from my account, something I hoped might make it possible to cleanly begin a new subscription.

I'd be grateful for any help. And if there is a solution for this that I missed, I'd suggest a corresponding update to Github Copilot's billing docs.

**Comment by githubpsyche** [comment, marked as answer]
Ah, I've realized that I'm actually still getting suggestions from Copilot. I thought I'd sufficiently tested for this before writing this post up but nope. I'm not sure how to delete/close this discussion given the clarification.

**Comment by otkrickey** [comment]
Solved: #43634
